apps/example-orchestrator-ui/components/Subscriptions/Subscriptions.tsx

```diff
@@ -0,0 +1,266 @@
+    useQueryWithGraphql,
+    getTypedFieldFromObject,
+    CheckmarkCircleFill,
+    MinusCircleOutline,
+    useOrchestratorTheme,
+    SubscriptionStatusBadge,
+    ControlColumn,
+    PlusCircleFill,
+} from '@orchestrator-ui/orchestrator-ui-components';
+import React, { FC } from 'react';
+import {
+    MyBaseSubscriptionEdge,
+    PythiaSortOrder,
+    SubscriptionGridQuery,
+    SubscriptionsSort,
+} from '../../__generated__/graphql';
+import {
+    GET_SUBSCRIPTIONS_PAGINATED,
+    GET_SUBSCRIPTIONS_PAGINATED_DEFAULT_VARIABLES,
+} from './subscriptionsQuery';
+import { useRouter } from 'next/router';
+import Link from 'next/link';
+import { EuiFlexItem } from '@elastic/eui';
+
+type Subscription = {
+    subscriptionId: string;
+    description: string;
+    status: string;
+    insync: boolean;
+    startDate: string;
+    endDate: string;
+    productName: string;
+    tag: string;
+    organisationName: string;
+    organisationAbbreviation: string;
+    notes: string;
+};
+
+type SubscriptionsProps = {
+    pageSize: number;
+    setPageSize: (updatedPageSize: number) => void;
+    pageIndex: number;
+    setPageIndex: (updatedPageIndex: number) => void;
+    sortOrder: SubscriptionsSort;
+    setSortOrder: (updatedSortOrder: SubscriptionsSort) => void;
+};
+
+export const Subscriptions: FC<SubscriptionsProps> = ({
+    pageSize,
+    pageIndex,
+    sortOrder,
+    setPageSize,
+    setPageIndex,
+    setSortOrder,
+}) => {
+    const router = useRouter();
+    const { theme } = useOrchestratorTheme();
+
+    const tableColumnConfig: TableColumns<Subscription> = {
+        description: {
+            displayAsText: 'Description',
+            initialWidth: 400,
+            renderCell: (cellValue, row) => (
+                <Link href={`/subscriptions/${row.subscriptionId}`}>
+                    {cellValue}
+                </Link>
+            ),
+        },
+        insync: {
+            displayAsText: 'In Sync',
+            initialWidth: 110,
+            renderCell: (cellValue) =>
+                cellValue ? (
+                    <CheckmarkCircleFill color={theme.colors.primary} />
+                ) : (
+                    <MinusCircleOutline color={theme.colors.mediumShade} />
+                ),
+        },
+        organisationName: {
+            displayAsText: 'Customer Name',
+            isHiddenByDefault: true,
+        },
+        organisationAbbreviation: {
+            displayAsText: 'Customer',
+            initialWidth: 200,
+        },
+        productName: {
+            displayAsText: 'Product',
+            initialWidth: 250,
+            isHiddenByDefault: true,
+        },
+        tag: {
+            initialWidth: 100,
+            displayAsText: 'Tag',
+        },
+        startDate: {
+            displayAsText: 'Start Date',
+            initialWidth: 150,
+            renderCell: (cellValue) =>
+                cellValue
+                    ? new Date(parseInt(cellValue) * 1000).toLocaleString(
```

Comment:
is this timezone aware, e.g. CEST for us?

Reply:
Added a todo in the code for now

Reply:
It's probably not localized :) Would be good to ticketify this todo so we don't find out about it next winter